src/pages/images/ImageSelector.tsx

```diff
@@ -81,17 +81,20 @@ const ImageSelector: FC<Props> = ({ onSelect, onClose }) => {
   const { data: canonicalImages = [], isLoading: isCiLoading } = useQuery({
     queryKey: [queryKeys.images, canonicalServer],
     queryFn: () => loadImages(canonicalJson, canonicalServer),
+    retry: false, // avoid retry to ease experience in airgapped deployments
```

Comment:
Is this maybe not too drastic? not sure if there is a better way though, since ideally we should be aware if we are in an airgapped environment.

Reply:
I want to avoid the complexity of another configuration key. I think the UI should just work in airgap without needing to do anything by user or admin.